apps/desktop-wallet/src/contexts/walletconnect.tsx

```diff
@@ -406,10 +414,13 @@ export const WalletConnectContextProvider: FC = ({ children }) => {
     console.log('👉 ARGS:', args)
   }, [])
 
+  const shouldInitialize = walletConnectClientStatus === 'initialization-failed'
+  useInterval(initializeWalletConnectClient, 3000, !shouldInitialize)
```

Comment:
Same here. This seems not used.

Reply:
I simplified the init code to only call `initializeWalletConnectClient` from one place.

I'll work on now to add a 10 retry limit.